engines/osxnative/resultset_test.go

```diff
@@ -106,11 +106,11 @@ func TestExtractFolder(t *testing.T) {
 
 	err = r.ExtractFolder("test-data", func(p string, stream ioext.ReadSeekCloser) error {
 		expected := path.Base(p) + "\n"
-		data, err := ioutil.ReadAll(stream)
+		data, err2 := ioutil.ReadAll(stream)
 		sdata := string(data)
 
-		if err != nil {
-			return err
+		if err2 != nil {
+			return err2
```

Comment:
Yes, but if I don't rename the variable in `data, err2 := ioutil.ReadAll(stream)` line 109 (where the rename is) then go-metalinter will complain.
The reason is that `err` will overshadow `err` from outer scope.

It's a decent warning to have as we could easily make a mistake by assuming the inner err is the same as outer err. We can also disable the overshadowing rule, but I suspect it'll catch a lot of small bugs for us...